OTKit/otkit-icons/ic_warning_timeline.svg

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <svg width="24px" height="24px" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-    <defs>
+    <g id="icon/line/ic_checkmark/Light-UI---Default" stroke="none" stroke-width="1" fill="#141A26" fill-rule="evenodd">
```

Comment:
same comment above

Reply:
We can simplify the svg so that svgo cleans up the id. That's what im doing here

Reply:
but id here is wrong. can you remove that?